lua/metals/config.lua

```diff
@@ -130,6 +130,7 @@ local valid_metals_settings = {
   "showImplicitConversionsAndClasses",
   "showInferredType",
   "superMethodLensesEnabled",
+  "testUserInterface",
```

Comment:
I'll comment here because I can't up above. But I think we also need to add `testExplorerProvider = debugging_provider` like we do with `debuggingProvider`. This way it will ensure dap is available and turn this on when it is.